hyperspy/tests/io/test_emd.py

```diff
@@ -63,6 +63,18 @@ def test_metadata():
     nt.assert_is_instance(signal, Image)
 
 
+class TestSaveSimpleSignal():
+
+    def test_save_simple_signal(self):
+        simple_signal = Signal(data_save)
+        simple_signal.save(
+                os.path.join(
+                    my_path, 'emd_files', 'example_temp.emd'), overwrite=True)
```

Comment:
Do you only want to check that the saving succeeds? I would probably have a look that the saved file has only the required fields defined, though it's up to you